prow/config.yaml

```diff
@@ -243,6 +245,7 @@ tide:
     istio/istio: squash
     istio/operator: squash
     istio/pkg: squash
+    istio/api: squash
```

Comment:
FYI, if all `istio` repos should use squash it is sufficient to specify `istio: squash` instead of enumerating each repo.

Reply:
Yeah, we'll get there... :-)